### DISCUSSION TITLE
Automation falsely said my username and password was in a commit

### DISCUSSION THREAD

**Discussion by crawsome** [community/community#145650] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Github support, I need help deciphering this email I got at 3AM last night from your automation.

>Your username and password were exposed in a commit to a public repository.

That woke me up and set me into a frenzy. I frantically looked to see but I couldn't find what Github was claiming. 

I think this was sent in-error. I cannot find my credentials in this commit. I see my old email as it was the old committer account name, but there's no password in the commit as the email claims.

Additionally, this commit is from 2017... why am I get notified about it now?

Can someone from support please reach out and demystify what Github was trying to tell me? 

I couldn't find another way to contact support, all the links are linking to documentation even though I click "Contact support" so I'm posting here.

Kind regards,

**Comment by samus-aran** [comment]
Hi @crawsome! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support (accounts) page](https://support.github.com/contact/account). Please open a ticket with them :)
